api/py/test/sample/group_bys/unit_test/entity_sample_group_by.py

```diff
@@ -0,0 +1,33 @@
+#     Copyright (C) 2023 The Chronon Authors.
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+from ai.chronon.group_by import Aggregation, GroupBy, Operation
+from sources import test_sources
+
+v1 = GroupBy(
+    sources=test_sources.staging_entities,
+    keys=["s2CellId", "place_id"],
```

Comment:
nit: i think we can use something that's a bit more generic

Reply:
I used existing group_by examples from `sample_team/group_bys/sample_group_by.py` and simplified them. We can separately make all examples configurations under sample_team more generic.